source/configuration_manual/acl.rst

```diff
@@ -276,6 +282,11 @@ Prevent all users from deleting their Spam folder (notice no x flag)
 
   INBOX.Spam owner lrwstipeka
 
+Allow a masteruser full access to all mailboxes, except no access to INBOX:
+
+  * user=masteruser lrwstipekxa
+  INBOX -user=masteruser lrwstipekxa
```

Comment:
Bullet list ends without a blank line; unexpected unindent.

This needs code-block.

Reply:
Oh, was too lazy to build it yesterday since I thought there can't be anything wrong in it.